zulipterminal/core.py

```diff
@@ -24,6 +24,10 @@ class Controller:
 
     def __init__(self, config_file: str, theme: ThemeSpec,
                  autohide: bool) -> None:
+
+        # Is False when ZT's UI has been loaded without any errors.
+        self.loading = True  # type: bool
```

Comment:
I would make this variable name more specific - not only does naming something this simply make it difficult to know where it might be referenced, but using eg. `self.ui_loaded = False` or `self.ui_loaded_successfully` (note the inverse case) probably makes the comments unnecessary?

Reply:
Yeah, makes sense